lib/bootloader_setup.pm

```diff
@@ -88,14 +88,16 @@ sub bootmenu_default_params {
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
         type_string " video=hyperv_fb:1024x768";
     }
+    return if is_jeos || is_casp('VMX');
```

Comment:
Y2debug isn't only about producing debug info for the installer

And I imagine we need to test yast inside jeos, maybe even in some small scale for yast also (I am surprised if there is absolutely NO yast in Caasp and that it will stay that way)